File: tests/codebase/fixtures/sample.php
<?php
namespace App\Service;

use App\Repo\UserRepo;
use App\Util\Foo;

class Greeter {
    public function greet(string $name): string { return $this->helper(1); }
    public function helper(int $x): string { return "hi"; }
}

interface Animal { public function name(): string; }

function helper(int $x): int { return $x + 1; }

helper(1);
helper(2);

File: tests/codebase/fixtures/sample.go
package main

import (
	"fmt"
	"os"
)

type Person struct {
	Name string
}

func (p *Person) Greet() string { return helper(p.Name) }

func helper(s string) string { return "hi " + s }

func main() {
	_ = os.Args
	_ = fmt.Sprintln("")
	p := &Person{Name: "x"}
	p.Greet()
	helper("a"); helper("b")
}
